packages/time-picker/src/vaadin-time-picker-item.js

```diff
@@ -3,11 +3,12 @@
  * Copyright (c) 2018 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css, html, LitElement } from 'lit';
+import { html, LitElement } from 'lit';
 import { ComboBoxItemMixin } from '@vaadin/combo-box/src/vaadin-combo-box-item-mixin.js';
 import { defineCustomElement } from '@vaadin/component-base/src/define.js';
 import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
 import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { itemStyles } from '@vaadin/item/src/vaadin-item-core-styles.js';
```

Comment:
I'm not sure if these styles should be classified as "core" since they are relevant to both Base and Lumo styles.

Reply:
This import is intentional, there are base styles for item in `vaadin-item-base-styles.js`